### ISSUE TITLE
Gradle is unable to download sources on version 1.4.1

### ISSUE THREAD

**Issue by AndreThlckr** [KasperskyLab/Kaspresso#370]
When using newer versions of the library, Gradle is unable to find and download the library sources (which allow you to read and debug the source code directly on the IDE).

## Expected Behavior

Projects with the Kaspresso dependency should be able to download sources.

## Actual Behavior

When trying do download sources, the following error appears:

```console
* What went wrong:
Execution failed for task ':app:DownloadSources'.
> Could not resolve all files for configuration ':app:downloadSources_10c6f7e9-408b-4f6a-8bd9-fe15e255981e'.
   > Could not find com.kaspersky.android-components:kaspresso:1.4.1@aar.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/kaspersky/android-components/kaspresso/1.4.1@aar/kaspresso-1.4.1@aar.pom
       - https://repo.maven.apache.org/maven2/com/kaspersky/android-components/kaspresso/1.4.1@aar/kaspresso-1.4.1@aar.pom
     Required by:
         project :app
```

## Steps to Reproduce the Problem

  1. Create an empty project;
  2. Add the dependency `androidTestImplementation "com.kaspersky.android-components:kaspresso:1.4.1"`;
  3. Create a test using classes from Kaspresso;
  4. Try to access the source (on IntelliJ IDE, `Ctrl+left click` a Kaspresso class name/method call);
  5. You will only be able to see the decompiled bytecode.

## Specifications

  - Library version: at least >= 1.4.1
  - IDE used: Android Studio

## Observations
  - I haven't tested on all versions, but sources were able to be downloaded at least up to version 1.2.1.

**Comment by Nikitae57** [comment]
Hi, @AndreThlckr! Thank you for your submission. Issue confirmed. We'll work on resolving it shortly and give you an update as soon, as fix would be ready

**Comment by AzamatCherchesov** [comment]
Hi, @AndreThlckr! The fix will be available in the next release. Thanks for the issue!

**Comment by Nikitae57** [comment]
Hello, again @AndreThlckr! We fixed this issue in the 1.5.1 release. Please, feel free to re-open the issue if you think that your problem isn't resolved
